templates/cube/microcerts.html

```diff
@@ -0,0 +1,379 @@
+      <div class="p-cube-progression is-networking {% if modules[7].status != 'passed' %}is-faded--left{% endif %}"></div>
+      <div class="p-cube-progression is-kernel {% if modules[4].status != 'passed' %}is-faded{% endif %}"></div>
+      <div class="p-cube-progression is-microk8s {% if modules[6].status != 'passed' %}is-faded--right{% endif %}"></div>
+      <div class="p-cube-progression is-virtualisation {% if modules[14].status != 'passed' %}is-faded--left{% endif %}"></div>
+      <div class="p-cube-progression is-storage {% if modules[11].status != 'passed' %}is-faded{% endif %}"></div>
+      <div class="p-cube-progression is-juju {% if modules[3].status != 'passed' %}is-faded--right{% endif %}"></div>
+      <div class="p-cube-progression is-maas {% if modules[5].status != 'passed' %}is-faded--left{% endif %}"></div>
+    </div>
+  </div>
+</section>
+
+{% if certified_badge %}
+
+<section class="p-strip--light is-slanted--top-right">
+  <div class="row">
+    <div class="col-3 col-start-large-2 u-hide--small">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/17541ecf-CUBE+certified+-+132x190_3.svg",
+        alt="CUBE certified",
+        width="175",
+        height="252",
+        hi_def=True,
+        loading="auto"
+        ) | safe
+      }}
+    </div>
+    <div class="col-7">
+      <h2>Badge of honor</h2>
+      <p style="margin-bottom:0.5rem;">Share your certification with your network</p>
+      <ul class="p-inline-list-icons u-no-padding--left u-no-margin--left u-no-padding--top">
+        <li class="p-inline-list__item">
+          <a title="Share on Twitter" href="https://twitter.com/share?text=I am officially a Certified Ubuntu Engineer. My CUBE 2020 from @Canonical means that I’m a proven @Ubuntu professional. Learn more about CUBE 2020 at https://www.ubuntu.com/cube&amp;url={{ certified_badge.image }}&amp;hashtags=CUBE2020">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/6d51a38d-Twitter.png",
+            alt="",
+            width="61",
+            height="20",
+            hi_def=True,
+            loading="auto"
+            ) | safe
+          }}
+          </a>
+        </li>
+        <li class="p-inline-list__item">
+          <a title="Share on LinkedIn" href="https://www.linkedin.com/shareArticle?url={{ certified_badge.image }}&title=CUBE 2020">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/72040dde-LindedIn.png",
+            alt="",
+            width="60",
+            height="20",
+            hi_def=True,
+            loading="auto"
+            ) | safe
+          }}
+        </a>
+        </li>
+      </ul>
+      <p>You can <a href="https://www.linkedin.com/profile/add?startTask=CERTIFICATION_NAME&name=CUBE 2020&organizationName=Canonical&certUrl={{ certified_badge.image }}">add this badge</a> to the Certification section of your LinkedIn profile.</p>
```

Comment:
Should CERTIFICATION_NAME be a variable?

Reply:
No, See [here](https://addtoprofile.linkedin.com/) for instructions followed